src/scripts/cancel-or-discard.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+source $BASH_ENV;
+
+echo "Checking for previous runs to clear for $TF_ORG_NAME/$TF_WORKSPACE_NAME..."
+
+# Get 50 most recent runs to check through
+RESPONSE_RUNS=$(curl -s \
+  --header "Authorization: Bearer $TERRAFORM_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  https://app.terraform.io/api/v2/workspaces/$TF_WORKSPACE_ID/runs?page%5Bsize%5D=50)
+
+#echo $RESPONSE_RUNS | jq 'select(.data[].attributes.status | IN(["pending","plan_queued","planning","planned","cost_estimating","cost_estimated","policy_checking","policy_checked"])) | .data[].id'
+
+# Filter the response to run IDs triggered by CCI that would be holding back future runs
+declare -a runs=$(
```

Comment:
**Praise**: Neato! Didn't realize `declare` was a thing!

Reply:
stack overflow taught me this one 😄